### PR TITLE
{bio}[goolf/1.7.20] BCFtools v1.3.1

### PR DESCRIPTION
**Pull request by pescobar**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@pescobar Travis doesn't agree:

```
EasyBuildError: 'Irresolvable dependencies encountered: GSL/1.16-goolf-1.7.20'
```

File: easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+source_urls = ['https://github.com/samtools/bcftools/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [
+	('zlib', '1.2.8'),
+	('GSL', '1.16'),
+]
+
+buildopts = " USE_GPL=1 GSL_LIBS='-lgsl -lopenblas'"
```
Comment (pescobar) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb]: I am thinking about doing `buildopts = " USE_GPL=1 GSL_LIBS='-lgsl -l${LIBBLAS}'"` but checking the build log I see `LIBBLAS set to -lopenblas -lgfortran`

I am not sure if the `-lgfortran` can break something. opinions? @boegel does easybuild provides any env var which contains only `-lopenblas` ? I couldn't find it in the build log..

Comment (boegel) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb]: @pescobar use `$LIBBLAS`, there's a reason `-lgfortran` is there (although it's only relevant when compiling Fortran, obviously)

It should be harmless when compiling non-Fortran code.

**Comment by pescobar** [comment]
@boegel I replace `-lopenblas` with `${LIBBLAS}`

It builds fine for me. can you test in your side?

File: easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+source_urls = ['https://github.com/samtools/bcftools/releases/download/%(version)s/']
+sources = [SOURCELOWER_TAR_BZ2]
+
+dependencies = [
+	('zlib', '1.2.8'),
+	('GSL', '1.16'),
+]
+
+buildopts = " USE_GPL=1 GSL_LIBS='-lgsl ${LIBBLAS}'"
```
Comment (pescobar) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb]: @boegel I did another test with `buildopts = " USE_GPL=1 GSL_LIBS='-lgsl -lgslcblas'"` and it also works :)

this one makes more sense for me, what do you think?

Comment (pescobar) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb]: btw, the defaults in the Makefile which don't work for me is `GSL_LIBS = -lgsl -lcblas`

Comment (boegel) [inline easybuild/easyconfigs/b/BCFtools/BCFtools-1.3.1-goolf-1.7.20.eb]: Well, I guess the one using `-lgslcblas` makes sense...

File: easybuild/easyconfigs/g/GSL/GSL-1.16-goolf-1.7.20.eb [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+name = 'GSL'
+version = '1.16'
+
+homepage = 'http://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting."""
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+toolchainopts = {'opt': True, 'optarch': True, 'unroll': True, 'pic': True}
```
Comment (boegel) [inline easybuild/easyconfigs/g/GSL/GSL-1.16-goolf-1.7.20.eb]: @pescobar first two are enabled by default, so drop those?

File: easybuild/easyconfigs/g/GSL/GSL-1.16-goolf-1.7.20.eb [hunk 4]
```diff
@@ -0,0 +1,18 @@
+easyblock = 'ConfigureMake'
+
+name = 'GSL'
+version = '1.16'
+
+homepage = 'http://www.gnu.org/software/gsl/'
+description = """The GNU Scientific Library (GSL) is a numerical library for C and C++ programmers.
+The library provides a wide range of mathematical routines such as random number generators, special functions and least-squares fitting."""
```
Comment (boegel) [inline easybuild/easyconfigs/g/GSL/GSL-1.16-goolf-1.7.20.eb]: line too long...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/6bc301e24b95acf189305700a4b3e149 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/7f1cf4eaa30d8173f07fbfc28a559abb for a full test report.

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**